test/dynamics/solvers/test_diffrax_DOP5.py

```diff
@@ -41,6 +41,23 @@ def simple_rhs(t, y):
 
         self.simple_rhs = simple_rhs
 
+    def test_no_t_eval(self):
+        """Test case for basic model without t_eval."""
+        t_span = np.array([0.0, 2.0])
+        y0 = jnp.array([1.0])
+
+        stepsize_controller = PIDController(rtol=1e-10, atol=1e-10)
+        results = diffrax_solver(
+            self.simple_rhs,
+            t_span,
+            y0,
+            method=Dopri5(),
+            t_eval=None,
+            stepsize_controller=stepsize_controller,
+        )
+
+        self.assertAllClose(results.t, t_span[-1])
```

Comment:
diffrax only returns the state at the final time if `t_eval` isn't specified?

Reply:
Yes, the default for diffrax is to just return the final time